attestation-service/verifier/src/sgx/mod.rs

```diff
@@ -43,22 +43,18 @@ pub struct SgxVerifier {}
 impl Verifier for SgxVerifier {
```

Comment:
Hi @mythi I've changed the API of verifier of `SGX`. PTL if it looks good to you.

BTW, it will now change the behavior of current CoCoAS.